bin/webpack.config.js

```diff
@@ -164,3 +154,22 @@ function assertRequiredOptionsArePresent(env) {
     throw new Error("'slideHeight' option is required but was not found");
   }
 }
+
+function composeMaterialVersion(material) {
+  const date = new Date().toISOString().substring(0, 10);
+
+  let commitHash = "";
+  try {
+    commitHash = childProcess
+      .execSync("git rev-parse --short HEAD", { cwd: material })
+      .toString();
```

Comment:
```suggestion
      .toString().trim();
```

To remove the LF from command output.